src/DIRAC/Core/Tornado/Server/TornadoREST.py

```diff
@@ -23,14 +23,20 @@ class TornadoREST(BaseRequestHandler):  # pylint: disable=abstract-method
     """Base class for all the endpoints handlers.
     It directly inherits from :py:class:`DIRAC.Core.Tornado.Server.BaseRequestHandler.BaseRequestHandler`
 
-    Each HTTP request is served by a new instance of this class.
-
     In order to create a handler for your service, it has to
     follow a certain skeleton::
 
       from DIRAC.Core.Tornado.Server.TornadoREST import TornadoREST
       class yourEndpointHandler(TornadoREST):
 
+        # Because a request to RESTful endpoint can be arbitrary,
```

Comment:
Because this class allows you to build sufficiently flexible APIs, the incoming request query path may not include the name of the system to which this API belongs.
So it is important to provide this information in a handler with `SYSTEM` variable.